pkg/test/application/echo/server/main.go

```diff
@@ -37,7 +37,19 @@ var (
 	key       string
 )
 
+var (
+	logOptionsFromCommandline = log.DefaultOptions()
+)
+
 func init() {
+	logOptionsFromCommandline.AttachFlags(
```

Comment:
You can use the cobra flags here directly.
We couldn't use Cobra flags in the test framework, since go test already depends on the standard flag parser.

Reply:
The echo application isn't currently using Cobra and I'd rather not switch over for this. Is this call incorrect?

Reply:
Not incorrect, but not necessary as well.  Should be straight-forward to add cobra support I think.